assets/js/modules/analytics/datastore/properties.js

```diff
@@ -407,6 +409,31 @@ const baseSelectors = {
 
 		return select( STORE_NAME ).isFetchingGetPropertiesProfiles( accountID );
 	} ),
+
+	/**
+	 * Gets all "classic" Universal Analytic and Google Analytics 4 (GA4) properties this account can access.
+	 *
+	 * Returns an array of all UA + GA4 analytics properties.
+	 *
+	 * Returns `undefined` if accounts have not yet loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state     Data store's state.
+	 * @param {string} accountID The Analytics Account ID to fetch properties for.
+	 * @return {(Array.<Object>|undefined)} An array of Analytics properties; `undefined` if not loaded.
+	 */
+	getPropertiesIncludingGA4: createRegistrySelector( ( select ) => ( state, accountID ) => {
+		let properties = select( STORE_NAME ).getProperties( accountID );
+
+		if ( select( MODULES_ANALYTICS_4 ) ) {
+			const	propertiesGA4 = select( MODULES_ANALYTICS_4 ).getProperties( accountID );
+			const propertiesGA4Mapped = propertiesGA4.map( ( property ) => ( { ...property, id: property._id } ) );
+			properties = properties.concat( propertiesGA4Mapped );
+		}
+
+		return sortBy( properties, [ 'name', 'id' ] );
```

Comment:
@danielgent i think it won't work as expected because GA4 properties store property names in the `displayName` attribute. Let's rewrite this to use custom compare function `properties.sort( ( a, b ) => { ... } )`.

Reply:
@eugene-manuilov so I can differentiate between the `ga` and `ua4` properties based on if they have the key `_id` or `id`, right? (from looking at the fixtures)

So I can do that.  :+1:   (and drop the `propertiesGA4Mapped` above)